backend/app/core/config.py

```
@@ -65,6 +68,25 @@ class Settings(BaseSettings):
+    # Phase 4: Payment Gateway Integration Point
+    payment_gateway_webhook_secret: str = ""
+
+    @model_validator(mode="after")
+    def require_secure_production_webhook_secret(self):
+        """Require a trimmed, minimum-length webhook secret in production."""
+        if self.environment.strip().lower() != "production":
+            return self
+
+        secret = self.payment_gateway_webhook_secret
+        if not secret or secret != secret.strip():
+            raise ValueError("payment gateway webhook secret is required in production")
```

Ruff (0.16.3)

[warning] Avoid specifying long messages outside the exception class

(TRY003)